src/briefcase/commands/convert.py

```diff
@@ -326,6 +326,23 @@ def input_url(self, app_name, override_value: str | None) -> str:
 
         return url
 
+    def input_console_app(self, override_value: str | None) -> bool:
+        """Ask about whether the app is a console application.
+
+        :returns: console_app
+        """
+        options = ["GUI", "Console"]
+        return (
+            self.select_option(
+                intro=("is this a GUI application or a Console application?"),
+                variable="Interface Style",
```

Comment:
These snippets are used as questions (or parts of responses), so capitalisation matters:
```suggestion
                intro="Is this a GUI application or a console application?",
                variable="interface style",
```